src/components/application_manager/src/plugin_manager/rpc_plugin_manager_impl.cc

```diff
@@ -77,6 +77,7 @@ RPCPluginManagerImpl::RPCPluginPtr RPCPluginManagerImpl::LoadPlugin(
   auto plugin_destroyer = [delete_plugin, plugin_dll](RPCPlugin* plugin) {
     LOG4CXX_DEBUG(logger_, "Delete plugin " << plugin->PluginName());
     delete_plugin(plugin);
+    DELETE_LOGGER();
```

Comment:
@SNiukalov why do you need to destroy logger on each plugin delete? Is there some relation? And which exactly instance of logger will be deleted?